### PR TITLE
docs: Installing Podman on Windows on a custom WSL distribution

### LINKED ISSUE #1035: Docs: Using a different/custom WSL2 distribution
### Is your feature request related to a problem? Please describe

In some context, certain users may want to use a different version than the fedora based image provided with Podman. It's possible to build its own podman-machine vm and use it for Qemu (on Mac) or WSL2 (windows). 

### Describe the solution you'd like

Provide How-to documentation on how one could be building his own image and configuring Podman Desktop so that it uses that image.

### Describe alternatives you've considered

_No response_

### Additional context

https://kenmoini.com/post/2022/05/podman-desktop-on-windows/#configuring-the-podman-desktop-wsl2-distribution

### PR DESCRIPTION
**Pull request by themr0c**
* added procedure: Installing Podman on Windows on a custom WSL distribution
* fixes https://github.com/containers/podman-desktop/issues/1035

Signed-off-by: Fabrice Flore-Thébault <ffloreth@redhat.com>

### CODE REVIEW

File: website/docs/Installation/windows-install/installing-podman-on-a-custom-wsl-distribution.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+     ## Mount the image
+     MNTPNT=$(buildah mount $CONTNR)
+
+     ## Enter the mount point
+     cd $MNTPNT
+
+     ## Create a tar file of the container filesystem, outside of the mounted filesystem
+     tar cvf /opt/my-linux.tar .
+     ```
```
Comment (themr0c) [inline website/docs/Installation/windows-install/installing-podman-on-a-custom-wsl-distribution.md]: I am wondering how deep we should go for this step. WDYT @benoitf @slemeur ?

File: website/docs/Installation/windows-install/installing-podman-on-a-custom-wsl-distribution.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    ```
+
+7. (Optionally) From your WSL distribution, [install Podman](https://podman.io/getting-started/installation#installing-on-linux) if required.
+
+8. From your WSL distribution, verify Podman is running:
+9. Configure Podman Desktop to use your WSL distribution. Edit the  `C:\Users\YOUR_USER_NAME\.config\containers\podman\machine\wsl\my-machine.json` file and set `ImagePath`.
+
+    ```
+    TODO
+    ```
```
Comment (themr0c) [inline website/docs/Installation/windows-install/installing-podman-on-a-custom-wsl-distribution.md]: This step still require research end refinement.

**Comment by benoitf** [comment]
I'm worried about that you manually edit some json files, then bypassing all `podman machine <commands>`

**Comment by themr0c** [comment]
I am wondering if there is a way to create the custom VM, and Podman Desktop detects that machine correctly?

**Comment by themr0c** [comment]
Second iteration:

* Run the entire procedure from the same Windows host (to not depend on another Linux host))
* Set as a prerequisite that the container image should be ready with Podman installed.

Haven't done real test yet.

File: website/docs/Installation/windows-install/installing-podman-on-a-custom-wsl-distribution.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+    ```powershell
+    > wsl
+    ```
+
+4. Run the commands from the default WSL distribution to export a tar file containing your Linux distribution from a container. Set the `<MYCONTAINER>` value for your container:
+
+     ```shell-session
+     $ sudo -i
+     # apt-get install -y buildah
```
Comment (odockal) [inline website/docs/Installation/windows-install/installing-podman-on-a-custom-wsl-distribution.md]: It seems that first we need to call `apt-get update`.

Comment (themr0c) [reply website/docs/Installation/windows-install/installing-podman-on-a-custom-wsl-distribution.md]: Yes. I just rewrote it using the default podman machine as a new distribution builder.

File: website/docs/Installation/windows-install/installing-podman-on-a-custom-wsl-distribution.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+5. Run your WSL distribution:
+
+    ```powershell
+    > wsl --distribution MyLinux
+    ```
+
+6. Register your WSL distribution as a Podman machine.
+
+// TODO
```
Comment (themr0c) [inline website/docs/Installation/windows-install/installing-podman-on-a-custom-wsl-distribution.md:66]: If I understand well, it's not sufficient to have the VM running: the `podman.exe` binary must be able to recognize it as a valid machine, if I understand well.

**Comment by benoitf** [comment]
closing due to lack of activity

**Pull request closed without merging**